lib/Controller/SettingsController.php

```diff
@@ -74,6 +74,7 @@ public function impersonate(string $userId): JSONResponse {
 		}
 
 		if (!$this->groupManager->isAdmin($impersonator->getUID())
+				&& !$this->groupManager->isDelegatedAdmin($impersonator->getUID())
```

Comment:
```suggestion
			&& !$this->groupManager->isDelegatedAdmin($impersonator->getUID())
```

just one tab less :)